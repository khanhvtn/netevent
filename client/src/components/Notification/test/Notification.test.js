import React from 'react'
import { create } from 'react-test-renderer'
import expect from 'expect'

import SystemNotification from '../Notification'

describe('SystemNotification Component', () => {
    test('Matches the snapshot', () => {
        const systemNotification = create(<SystemNotification />)
        expect(systemNotification.toJSON()).toMatchSnapshot()
    });
});