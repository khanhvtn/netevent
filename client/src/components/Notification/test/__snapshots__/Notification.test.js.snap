// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SystemNotification Component Matches the snapshot 1`] = `<div />`;
